cli/internal/context/testdata/duplicate-pkgs/package.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "repo-with-dupe-workspaces",
+  "workspaces": [
+    "apps/**"
```

Just in case anybody models after this we want to avoid `**`
```suggestion
    "apps/*"
```

---

updated. but why?